src/vs/workbench/services/runtimeStartup/common/runtimeStartup.ts

```diff
@@ -166,11 +166,18 @@ export class RuntimeStartupService extends Disposable implements IRuntimeStartup
 		// auto-start a runtime.
 		this._register(this.onDidChangeRuntimeStartupPhase(phase => {
 			if (phase === RuntimeStartupPhase.Complete) {
-				if (!this.hasAffiliatedRuntime() &&
+				// if no runtimes were found, notify the user about the problem
+				if (this._languageRuntimeService.registeredRuntimes.length === 0) {
+					this._notificationService.error(nls.localize('positron.runtimeStartupService.noRuntimesMessage',
+						"No runtimes found. Please see the [Get Started](https://positron.posit.co/start.html) \
+					page for instructions on how to add a runtime."));
```

Comment:
```suggestion
						"No interpreters found. Please see the [Get Started](https://positron.posit.co/start) \
						documentation to learn how to prepare your Python and/or R environments to work with Positron."));
```

Reply:
We try to use "interpreters" in publicly facing UI (even though all the code internally says "runtime") and folks may still be confused about what that means so let's go ahead and mention Python and R in this message.